dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -77,7 +77,11 @@ dependent_startup_wait_for=zebra:running
 {% endif %}
 
 [program:bgpd]
+{% if FEATURE.bmp is defined and FEATURE.bmp.state is defined and FEATURE.bmp.state == "enabled" %}
```

Comment:
>FEATURE [](http://example.com/codeflow?start=6&length=7)

Could you make your 2 if statement consistent? Why did not you check "FEATURE is defined" here?

Reply:
done, previously just follow other below section style.